docs/plugins/repository-shared-settings.asciidoc

```diff
@@ -1,11 +1,12 @@
 `max_restore_bytes_per_sec`::
 
-    Throttles per node restore rate. Defaults to `40mb` per second.
+    Throttles per node restore rate. Defaults to unlimited.
+    Note that restores are also throttled through <<recovery,recovery settings>>.
```

Comment:
This file is used in the Plugins and Integrations book, which is separate from the Elasticsearch Reference. Because an internal xref is used, the system assumes the reference exists in Plugins and Integrations.

You can fix this by using an external link instead:
```suggestion
    Note that restores are also throttled through {ref}/recovery.html[recovery settings].
```

Reply:
Thank you so much!

Reply:
Happy to help!